fastagency/api/openapi/client.py

```diff
@@ -129,19 +134,22 @@ def _get_security_params(self, name: str) -> Optional[BaseSecurityParameters]:
                 )
 
         # check if security matches security parameters
```

Comment:
why don't you refactor this into a function that can be independently tested?

Reply:
Done